credentials/credential-tsm-protocol.md

```diff
@@ -109,41 +122,71 @@ More information about the verification process can be found in the [TSM Verific
 
 ## 2.2. Risk Assessment
-TBD
+[The Mining Association of Canada (MAC) Board of Directors (Board)](https://mining.ca/about-us/board-of-directors/) is the the primary authority for the risk assessment process of the TSM Programme. The Board relies heavily on the collective expertise and advice from the [Community of Interest (COI) Panel](https://mining.ca/towards-sustainable-mining/how-tsm-works/community-of-interest-advisory-panel/). Key issues are identified and presented to the TSM COI Panel. 
 
 ## 2.3. Trust Assurance and Certification
-TBD
+  
+Trust Assurance and Certification is determined primarily by the [MAC Board of Directors](https://mining.ca/about-us/board-of-directors/). The [TSM Community of Interest (COI) Panels](https://mining.ca/towards-sustainable-mining/how-tsm-works/community-of-interest-advisory-panel/) is an independent, multi-interest group comprised of about 12 to 15 individuals from Aboriginal groups, communities where the industry is active, environmental and social NGOs, and labor and financial organizations that advises the MAC Board of Directors.
+
+MAC utilizes the **MUTUAL COOPERATION AND LICENSING AGREEMENT RESPECTING THE USE AND APPLICATION OF TSM** as a trust assurance framework for international adoption. MAC has identified the following minimum categories of requirements for Licensees: 
+1.	Guiding Principles
+2.	Performance Indicators
+3.	Facility-level Reporting
+4.	Independent Verification
+5.	Public Reporting of Facility Level Reporting 
+6.	Community of Interest Advisory Body
+7.	Condition of Membership
 
 ## 2.4. Governance Requirements
 The TSM Protocol Credential is governed by the [**TSM Program Governance**](https://mining.ca/wp-content/uploads/dlm_uploads/2021/12/TSM-Program-Governance.pdf) 
 
 ## 2.5. Business Requirements
 
-The primary use of the TSM Protocol Credential is verification of TSM scores, see [Mining Association of Canada (MAC) Verifier Governance Framework](https://github.com/bcgov/bc-vcpedia/blob/TSM/applications/application-tsm-mac-verification.md)
+The primary use of the TSM Protocol Credential is verification of TSM scores, see [Mining Association of Canada (MAC) Verifier Governance Framework](https://mining.ca/towards-sustainable-mining/external-verification/#:~:text=Individuals%20interested%20in%20becoming%20TSM%20verifiers%20should%20confirm,Mining%20Association%20of%20Canada%E2%80%99s%20annual%20verifier%20training%20workshop.)
 
 ## 2.6. Technical Requirements (Credential)
 
-The Verifiable Credential format for this credential is AnonCreds specification (https://anoncreds-wg.github.io/anoncreds-spec/)
+The Verifiable Credential format for this credential is AnonCreds specification (https://hyperledger.github.io/anoncreds-spec/)
 
 ### 2.6.1 Schema Definition
 
-__Schema Name:__ tsm.protocol
+__Schema Name:__ tsm.canadian.summary.report
```

Comment:
I'm not sure this is specific to Canada. I think this is still representative for TSM internationally. So, maybe call it tsm.summary.report

If this really is only for Canada, many more changes are required to this GF

Reply:
changed